.github/workflows/doctor.yml

```diff
@@ -8,20 +8,18 @@ on:
       - Library/Homebrew/extend/os/diagnostic.rb
       - Library/Homebrew/extend/os/mac/diagnostic.rb
       - Library/Homebrew/os/mac/xcode.rb
-    branches-ignore:
-      - master
 env:
   HOMEBREW_DEVELOPER: 1
   HOMEBREW_NO_AUTO_UPDATE: 1
 jobs:
   tests:
     strategy:
       matrix:
-        version: [10.15, 10.14, 10.13]
+        version: ['11-arm', '11.0', '10.15', '10.14']
```

Comment:
Would be update `11-arm` at some point here to be `11-arm64` CC @jonchang

Reply:
In that case, it'll be worth renaming `11.0` to `11-x86_64` or something since it isn't actually 11.0 anymore.